builtin/logical/pki/path_intermediate.go

```diff
@@ -179,15 +170,6 @@ func (b *backend) pathSetSignedIntermediate(
 		return nil, fmt.Errorf("saved key could not be parsed successfully")
 	}
 
-	equal, err := certutil.ComparePublicKeys(parsedCB.PrivateKey.Public(), inputBundle.Certificate.PublicKey)
```

Comment:
What's the reason for removing this? This is ensuring that the certificate that you're setting matches the backend's stored key. Seems like a decent check to have.